### PR TITLE
[GLUTEN-1515][CH] Support agg functions bit_and/bit_or/bit_xor

### PR DESCRIPTION
**Pull request by exmy**
## What changes were proposed in this pull request?

(Fixes: \#ISSUE-1515)

## How was this patch tested?

UT

### CODE REVIEW

**Comment by exmy** [comment]
related CH change [ClickHouse#49292](https://github.com/ClickHouse/ClickHouse/pull/49292). Let's wait until it's ready.

File: gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala [hunk 1]
```diff
@@ -39,8 +39,8 @@ object ExpressionMappings {
   final val BLOOM_FILTER_AGG = "bloom_filter_agg"
   final val VAR_SAMP = "var_samp"
   final val VAR_POP = "var_pop"
-  final val BIT_AND_AGG = "bitwise_and_agg"
-  final val BIT_OR_AGG = "bitwise_or_agg"
+  final val BIT_AND_AGG = "bit_and"
+  final val BIT_OR_AGG = "bit_or"
```
Comment (exmy) [inline gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala:43]: @Yohahaha It seems that the Velox UT failed due to these changes. "Validation failed due to bit_and was not supported in AggregateRel."
Could we also align `BIT_AND_AGG` and `BIT_OR_AGG` to conform with Spark nodeName?

Comment (Yohahaha) [reply gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala:43]: Gluten + Velox reuse some Prestosql's functions currently, unfortunately bitwise_or_agg is that function.

I think Velox backend and CH backend should use different function mapping list in their own module, CC @zhouyuan @FelixYBW

Comment (Yohahaha) [reply gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala:43]: Anyway, let's open a issue for this.

Comment (exmy) [reply gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala:43]: OK, I see. Is it possible to support it in the Velox backend by registering aliases, while still keeping unified function mapping list in gluten-core?

**Comment by Yohahaha** [comment]
PR #1550 has merged, please rebase and retry.

**Review by zzcclp — APPROVED** [review]
LGTM

**Pull request merged**